cmd/ttn-lw-cli/commands/end_device_templates.go

```diff
@@ -310,10 +323,15 @@ This command takes end device templates from stdin.`,
 			if err != nil {
 				return err
 			}
-			stream, err := ttnpb.NewEndDeviceTemplateConverterClient(dtc).Convert(ctx, &ttnpb.ConvertEndDeviceTemplateRequest{
-				FormatId: formatID,
-				Data:     data,
-			})
+
+			req := &ttnpb.ConvertEndDeviceTemplateRequest{
+				FormatId:            formatID,
+				Data:                data,
+				EndDeviceVersionIds: &ttnpb.EndDeviceVersionIdentifiers{},
+			}
+			// The execution should not stop if the flags are not found, therefore the error is ignored.
+			_, _ = req.EndDeviceVersionIds.SetFromFlags(cmd.Flags(), "end-device-version-ids")
```

Comment:
nit; there's a difference between the flags not set (not changed) and failing to parse flags (i.e. wrong EUI length)

Reply:
True, changed it so that it there is an error it will return it. Indeed the flags should always be there for this command in specific since it is something declared at:
```
endDeviceTemplatesFromDataCommand.Flags().AddFlagSet(templateFormatIDFlags())
	endDeviceTemplatesFromDataCommand.Flags().AddFlagSet(dataFlags("", ""))
	ttnpb.AddSetFlagsForEndDeviceVersionIdentifiers(
		endDeviceTemplatesFromDataCommand.Flags(),
		"end-device-version-ids",
		false,
	)
```